ecl/hql/hqlgram.y

```diff
@@ -7779,6 +7868,11 @@ simpleDataSet
                             $$.setExpr(createDataset(no_rowsetindex, $1.getExpr(), $3.getExpr()));
                             $$.setPosition($1);
                         }
+    // Prefix form of the filter
+    | '(' '+' ')' '(' dataSetOrRowList optAppendAttrs ')'
```

Comment:
I was wondering whether supporting the shorthand (&)(a,b) might allow for some intuition from users but it may also confuse things when used with options?